charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -62,6 +62,15 @@ data:
   {{- end }}
   {{ .Values.server.coordinatorExtraConfig | indent 4 }}
 
+{{- if .Values.accessControl }}{{- if eq .Values.accessControl.type "configmap" }}
+  access-control.properties: |
+    access-control.name=file
```

Comment:
Eventually it might be useful to support other types of access control plugins as well. But that's for a separate PR.